src/modules/flow/int/int.c

```diff
@@ -212,6 +215,32 @@ reset_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t co
         &mdata->val);
 }
 
+static int
+set_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id,
+    const struct sol_flow_packet *packet)
+{
+    int r;
+    int32_t value;
+    struct accumulator_data *mdata = data;
+
+    r = sol_flow_packet_get_irange_value(packet, &value);
+    SOL_INT_CHECK(r, < 0, r);
+
+    if (value == mdata->val.val) return 0;
```

Comment:
what if we ignored the initial packet and value is the same as that? Should we send a packet or not?